tests/test_marker_type.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from cv2 import aruco
+
+from zoloto.marker_type import MarkerType
+
+EXPECTED_marker_typeS = {
```

Comment:
```suggestion
EXPECTED_MARKER_TYPES = {
```

Same for the rest of this file

Reply:
Guess I forgot to do case-sensitive find/replace :man_facepalming: